pkg/kubectl/cmd/edit.go

```diff
@@ -120,12 +120,18 @@ func RunEdit(f *cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args
 		return err
 	}
 
+	// NB: we use `f.Decoder(false)` to get a plain deserializer for
```

Comment:
I don't think that's an issue. If the comment isn't living right next to this line, I could foresee someone changing it back to true without noticing the comment.